ffi/koptcontext.lua

```diff
@@ -15,6 +15,34 @@ else
     k2pdfopt = ffi.load("libs/libk2pdfopt.so.2")
 end
 
+--[[
+Leptonica cheatsheet:
+-- Data structures:
+PIX - basic data structure - stores image
+PIXA - array of PIX
+BOX - stores rectangle (x, y, w, h)
+BOXA - array of BOX
+SEL - selector for morphological operations
+SELA - array of SEL
+-- Functions:
+boxaGetCount(boxa) - get number of elements in boxa array
+boxaGetBox(boxa, index, flag) - flag can be C.L_COPY (creates new copy) or C.L_CLONE (returns ref-counted handle)
+pixThresholdToBinary(pix, threshold) - returns new black-white PIX, if the source pixel is < threshold,
+                                       resulting pixel is 1 (black), otherwise 0 (white)
+pixGetRegionsBinary(input_pix,
+                    output_halftonemask,
+                    output_textline_mask,
+                    output_textblock_mask,
+                    debug) - returns 0 on success and 1 on failure, output_* are PIX
+pixDestroy(pix) - free memory from PIX
+boxDestroy(box) - free memory from BOX
+selDestroy(sel) - free memory from SEL
+pixaDestroy(pixa) - free memory from PIXA
+boxaDestroy(boxa) - free memory from BOXA
+selaDestroy(sela) - free memory from SELA
+]]
```

Comment:
```suggestion

@module ffi.koptcontext
]]
```
See a couple of the other files around here for examples. You can generate the docs to see if they look as expected by running `ldoc .` in the /doc folder in front.
https://github.com/koreader/koreader-base/blob/3fdf18c280d821492abb51befb202fe96f08d1f5/ffi/MD5.lua#L1-L8